pkg/ngctl/cmd/cmd.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"flag"
+	"io"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/console"
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/info"
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/list"
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/options"
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/use"
+	cmdutil "github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/util"
+	"github.com/vesoft-inc/nebula-operator/pkg/ngctl/cmd/version"
+)
+
+const (
+	nkcLongDescription = `
+		"ngctl"(NebulaGraph Kubernetes Control) controls the Nebula cluster manager on Kubernetes.
+`
+)
+
+func NewNkctlCmd(in io.Reader, out, err io.Writer) *cobra.Command {
```

Comment:
rename this function like NewNgctlCmd will be better

Reply:
Emm, u r right!